diff --git a/pgtune b/pgtune
index 6e2934c..2f2db04 100755
--- a/pgtune
+++ b/pgtune
@@ -660,7 +660,7 @@ def read_options(program_args):
     parser.add_option('-D', '--debug', action="store_true", dest="debug",
                       default="False", help="Enable debugging mode")
 
-    parser.add_option('-S', '--settings', dest="settings_dir", default=None,
+    parser.add_option('-S', '--settings', dest="settings_dir", default="/usr/share/pgtune",
                       help="Directory where settings data files are located at.  Defaults to the directory where the script is being run from")
 
     parser.add_option('--doctest', help='run doctests', action='store_true')
